package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@bbp/nexus-sdk",
-  "version": "1.0.0-beta.20",
+  "version": "1.0.0",
```

Comment:
I don't think we should change packages versions in PRs like this. We should create a PR with a package bump after the tag/release process. 

Also, I don't think we've reached 1.0.0 yet

Reply:
Resolvers were the missing part before we cover v1.0 API. We already have views, schemas, realms...

Also, the next step would be to tackle AggregateSparqlViews which is a v1.1 feature, so we need to release v1.0.0 before

Reply:
We don't support cwrite operations on views and we don't support any permission based operation yet...

Reply:
So we are likely to never have a v1.0.0 release and skip right to the v1.1.0 one

Reply:
I guess we need to put more effort in SDK!